express-delete/index.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  const newArr = [];
+  for (const element in grades) {
+    newArr.push(grades[element]);
+  }
+  res.json(newArr);
+});
+
+app.delete('/api/grades/:id', (req, res) => {
+  delete grades[req.params.id];
+  res.sendStatus(204);
+});
+
+app.listen(3000, () => {
```

Comment:
You should always leave the console log in the `listen` callback so you can know that your server started successfully. You can use the following comment so the linter will ignore the console log:
```JAVASCRIPT
  // eslint-disable-next-line no-console
  console.log('Express server listening on port 3000');
```

Reply:
Ah, ok!  That's really good to know!  Thank you!